pkgs/tools/misc/otel-cli/default.nix

```diff
@@ -20,6 +20,8 @@ buildGoModule rec {
       --replace 'const minimumPath = `/bin:/usr/bin`' 'const minimumPath = `${lib.makeBinPath [ getent ]}`'
   '';
 
+  passthru.updateScript = nix-update-script {};
```

Comment:
personal curiosity, but what is this doing?

Reply:
it is https://github.com/Mic92/nix-update, showed up in tweags office hours. very convenient way to auto update a package version.

See https://www.youtube.com/live/fd11ihDmPNc?feature=share&t=2171

happy to remove if it aint primetime, but this is simple enough for version updates that it can help when being picked up by automated tools.

Reply:
no push back at all, just curious what it did; thanks for the pointer